tutorials/Appendix/Quick Topic Tutorials Notebooks/Quantum Oscillator Wavefunction Plot.ipynb

```diff
@@ -1,22 +1,24 @@
 {
  "cells": [
   {
-   "source": [
-    "# Demo Notebook for Plotting Wavefunctions of the Quantum Harmonic Oscillator (LC Circuit) "
-   ],
    "cell_type": "markdown",
-   "metadata": {}
+   "id": "reliable-developer",
```

Comment:
The id's in the notebook are "odd". 

`"id": "reliable-developer",`

It is throughout the notebook.